localisation/customizable_localisation/wc_customizable_localisation_generic_artifacts.txt

```diff
@@ -1,3 +1,83 @@
+# Common Origins
+defined_text = {
+	name = GetArtifactCommonOrigin
+
+	text = { localisation_key = artifact_common_origin_1 }
+	text = { localisation_key = artifact_common_origin_2 }
+	text = { localisation_key = artifact_common_origin_3 }
+	text = { localisation_key = artifact_common_origin_4 }
+	text = { localisation_key = artifact_common_origin_5 }
+	text = { localisation_key = artifact_common_origin_6 }
+	text = { localisation_key = artifact_common_origin_7 }
+	text = { localisation_key = artifact_common_origin_8 }
+	text = { localisation_key = artifact_common_origin_9 }
+	text = { localisation_key = artifact_common_origin_10 }
+}
```

Comment:
Just question. This works with names too or only with descriptions?

Reply:
It works only if there is no more than one random `defined_text`
Descriptions have one random `defined_text`, so it works in descriptions
But names have two random `defined_text`, so artifacts will be less random. For example, if artifact get prefix №3, it will always have suffix №7, etc.